docs/frequency-analysis.md

```diff
@@ -0,0 +1,11 @@
+Frequency analysis, which is also known as counting letters, is a useful study within the field of computer science. This is due to its function because it is being capable of outputting the frequency of letters or groups of letters in a ciphertext.
```

Comment:
Frequency analysis can be used to count letters, but in our case, we are counting the frequency of meaningful words and phrases and filtering out stopwords.